### PR TITLE
Improve the friendliness of code names for Util.java

### PR DESCRIPTION
**Pull request by utzcoz**
The `Util.java` is released to users. IMO, it's not a great practice that released code including words like "youSuckJava". I'm not familiar with slang, and please let me know if I understand wrong.

### CODE REVIEW

**Comment by utzcoz** [comment]
@hoisie self-hosted runner broken again.

**Review by hoisie — APPROVED** [review]
Lol nice

File: utils/src/main/java/org/robolectric/util/Util.java [hunk 1]
```diff
@@ -99,11 +99,11 @@ public static Path pathFrom(URL localArtifactUrl) {
   }
 
   public static List<Integer> intArrayToList(int[] ints) {
```
Comment (hoisie) [inline utils/src/main/java/org/robolectric/util/Util.java:101]: nit: can this just be removed? It's not used anywhere in the code.

**Comment by hoisie** [comment]
> self-hosted runner broken again.

I know, it's so annoying, see https://github.com/actions/runner/issues/805 :( Every time the runner version is bumped, I need to rebase manually in a custom branch, rebuild, and run set up each time :(

File: utils/src/main/java/org/robolectric/util/Util.java (hunk 1 above)
Comment (utzcoz) [reply utils/src/main/java/org/robolectric/util/Util.java:101]: We can mark it deprecated, and remove it at next release, because it is public to users.

Comment (utzcoz) [reply utils/src/main/java/org/robolectric/util/Util.java:101]: I will send a new PR to mark it deprecated this weekend.

**Comment by i-am-the-slime** [comment]
This looks like a regression to me. I think the previous code did a much better job of communicating that Java sucks.

**Comment by hoisie** [comment]
🤣

**Pull request merged**